copy/multiple.go

```diff
@@ -27,23 +32,90 @@ const (
 type instanceCopy struct {
 	op           instanceCopyKind
 	sourceDigest digest.Digest
+
+	// Fields which can be used by callers when operation
+	// is `instanceCopyClone`
+	compressionVariant            *compression.Algorithm
+	cloneAddPlatform              *imgspecv1.Platform
+	cloneAddAnnotations           map[string]string
+	cloneAddCompressionAlgorithms []compression.Algorithm
```

Comment:
Non-blocking: Maybe `s/cloneAdd/clone/`; “add” is a discriminator in `ListOpAdd`, which we use to implement `…Clone`; so saying both here does not help, and it can be potentially confusing (“add annotations” … on top of which pre-existing ones?).